File: segundo_punto.cpp
#include <iostream>

using namespace std;

int main() {
    //Declarar variable para descuentos
    double precio, descuento, preciofinal;

    //Solicitar el precio full
    cout <<"ingrese el valor del producto " ;
    cin >> precio;

    //calcular el 15 % de un producto
    descuento = precio * 0.15;
    preciofinal = precio - descuento;
   
    //mostrar resultados
    cout <<"Descuento aplicado: $" << descuento << endl;
    cout << "Precio final con el respectivo descuento: $" << preciofinal << endl;

    return 0;
}

File: segundo_punto.py
#Descuento a unproducto
numero = float(input("Dame el valor full del producto: $"))

descuento = numero * 0.15
precio_final = numero - descuento

#imprimir resultados
print (f"Descuento aplicado: ${descuento:.2f}")
print (f"Precio final con descuento: ${precio_final:.2f}")
